### PR TITLE
[1461] Add course salary edit page

### PR DESCRIPTION
**Pull request by dankmitchell**
### Context
Re-creating `/organisation/2cf/course/self/2yn6/salary` in rails

### Changes proposed in this pull request
Add `/salary` to a course

c# | rails
:--:|:--:
c#|rails

### CODE REVIEW

**Comment by dankmitchell** [comment]
[Add edit course length and salary page](https://trello.com/c/yaswoDLP/1461-add-edit-course-length-and-salary-page)

File: app/views/courses/salary.html.erb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_for course, url: publish_provider_course_path(course.provider.provider_code, course.course_code), method: :post do |f| %>
+      <div class="govuk-form-group">
+        <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+            <h2 class="govuk-fieldset__heading">Course length</h2>
+          </legend>
+            <div class="govuk-radios" data-module="radios">
```
Comment (fofr) [inline app/views/courses/salary.html.erb]: Should this be a partial that's shared with fees page?

File: app/views/courses/salary.html.erb [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<%= content_for :page_title, "#{course.name_and_code} - Course length and salary" %>
```
Comment (fofr) [inline app/views/courses/salary.html.erb]: Can we front-load the title with the most relevant bit?

Comment (dankmitchell) [reply app/views/courses/salary.html.erb]: Done

File: app/views/courses/salary.html.erb (hunk 1 above)
Comment (dankmitchell) [reply app/views/courses/salary.html.erb]: Good shout - Done.

File: app/views/courses/_course_length.html.erb [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<div class="govuk-form-group">
+  <fieldset class="govuk-fieldset">
+    <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+      <h2 class="govuk-fieldset__heading">Course length</h2>
+    </legend>
+      <div class="govuk-radios" data-module="radios">
```
Comment (tvararu) [inline app/views/courses/_course_length.html.erb]: Indentation, can ignore

Comment (dankmitchell) [reply app/views/courses/_course_length.html.erb]: Fixed 😊

**Pull request merged**